src/pages/student/categories/CategoryDetail/index.js

```diff
@@ -71,6 +92,20 @@ function Subcategories() {
     });
   };
 
+  const renderQuizList = () => {
+    return quizzes.map((quiz, idx) => {
+      return <QuizzesCard quiz={quiz} key={idx}/>;
+    });
+  };
+
+  function message(props) {
+    return (
+      <div className={style.noResultsMessage}>
+        <p className={style.message}>{props}</p>
+      </div>
+    );
+  }
+
```

Comment:
```suggestion
  const noResultMessage = (messageText) => {
    return (
      <div className={style.noResultsMessage}>
        <p className={style.message}>{messageText}</p>
      </div>
    );
  }

```

use ES6 function convention and update the function name as well as the argument being passed